### ISSUE TITLE
Lourdeurs avec un grand nombre d'utilisateurs

### ISSUE THREAD

**Issue by lpofredc** [PnX-SI/UsersHub#121]
la @lpoaura a fait le choix d'intégrer dans UsersHub les contributeurs VisioNature pour le rattachement des observateurs aux observations (env. 27000). Cela rend UsersHub inutilisable dans la page d'entrée de l'application (liste des utilisateurs) qui charge l'intégralité des utilisateurs.

Il pourrait être intéressant de privilégier un par API avec pagination (https://datatables.net/extensions/searchpanes/serverside.html).

**Comment by lpofredc** [comment]
Pour palier à ce problème avec un fix simple, on a fait le choix de patcher notre instance en filtrant les utilisateurs sur leur statut `active`. 

https://github.com/PnX-SI/UsersHub/compare/master...lpoaura:UsersHub:gnlpoaura

Si cela concerne d'autres, je peux proposer une PR avec une option pour activer ce filtrage.

**Comment by camillemonchicourt** [comment]
OK intéressant comme solution en effet.
Je dirai qu'il faudrait pouvoir faire en sorte que la liste des utilisateurs soit filtrée par défaut sur les ACTIF, mais pouvoir désactiver ce filtre pour pouvoir les afficher et y accéder.

**Comment by jacquesfize** [comment]
Je suis d'accord avec @lpofredc. Pour moi, la solution la plus adapté est de faire la pagination.
